network-api/networkapi/wagtailpages/pagemodels/customblocks/image_grid.py

```diff
@@ -2,8 +2,23 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
+class ImageGridAltText(blocks.StructValue):
+    """Get alt_text or default to image title."""
+
+    def alt(self):
+        alt_text = self.get('alt_text')
+        image = self.get('image')
+        if alt_text:
+            return alt_text
+        return image.title
+
+
 class ImageGrid(blocks.StructBlock):
     image = ImageChooserBlock()
+    alt_text = blocks.CharBlock(
+        required=False,
```

Comment:
This looks like it's not working. Do we need `blank=True` instead?

Reply:
hm, tried `blank=True` and `null=True`, but the CMS still goes "nope, I _need_ you to fill in an alt text"